decksite/data/rule.py

```diff
@@ -0,0 +1,189 @@
+from typing import List
+
+from decksite.data import deck
+from decksite.database import db
+from magic.models import Deck
+from shared.container import Container
+
+
+def num_classified_decks() -> int:
+    sql = 'SELECT COUNT(DISTINCT(deck_id)) AS c FROM ({apply_rules_query}) AS applied_rules'.format(apply_rules_query=apply_rules_query())
+    return (db().select(sql))[0]['c']
+
+def mistagged_decks() -> List[Deck]:
+    sql = """SELECT
+                    deck_id, rule_archetype.name AS rule_archetype_name, tagged_archetype.name AS tagged_archetype_name
+                FROM
+                    ({apply_rules_query}) AS applied_rules
+                JOIN
```

Comment:
We always explicitly say `INNER JOIN` or `LEFT JOIN`. Probably because I'm not actually sure what the default is. `INNER`? I'm not sure if that should be a standard practice or not. Would help me :P

Reply:
`INNER` is definitely default, but I don't mind being explicit